shape_files/places/make.inserts

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+
+require 'fileutils'
+require 'pathname'
+require 'zip'
+require_relative "../../app/models/grda_warehouse/us_census_api/state_fips"
+
+if ARGV[0].to_s.length != 2
```

Comment:
Any reason not to have this default to`ENV['RELEVANT_COC_STATE']`?

Reply:
It sort of does in "app/models/grda_warehouse/shape/installer.rb" where it's called.